### DISCUSSION TITLE
Editing cheat sheets

### DISCUSSION THREAD

**Discussion by parisi-n** [KDE/ghostwriter#705] [Q&A] [answered]
Dear writers,

I am using the Windows portable edition of the last Ghostwriter release.
Is there a way to edit and custom the cheat sheet attached to the program in order to show my custom tags?

Thank you

**Comment by mruiz42** [comment, marked as answer]
Not at the moment. They seem to be defined in [mainwindow.cpp:1322](https://github.com/wereturtle/ghostwriter/blob/master/src/mainwindow.cpp#L1322) 

I'm not sure how useful this might be since AFAIK markdown is parsed similarly across different interpreters. What use-case would it be useful to add this feature?

If you really wanted to do this you could just edit the code I mentioned above to include your custom tags, then build yourself.

**Reply by parisi-n** [reply]
Hi mruiz42,

thank you for your answer. This sounds as good idea, building my personal build with my custom markdown code.
